packages/next/types/global.d.ts

```diff
@@ -43,6 +43,11 @@ declare module '*.module.scss' {
   export default classes
 }
 
+declare module '*.css?raw' {
```

Comment:
Isn't t that file shipped to consumers? If we just need it once, we should just `@ts-expect-error` the single import